ale_linters/elixir/credo.vim

```diff
@@ -45,13 +45,24 @@ function! ale_linters#elixir#credo#GetMode() abort
     endif
 endfunction
 
+function! ale_linters#elixir#credo#GetConfigFile() abort
+    let l:config_file = get(g:, 'ale_elixir_credo_config_file', '')
+
+    if len(l:config_file) == 0
+        return ''
+    else
+        return ' --config-file ' . l:config_file
+    endif
```

Comment:
GitHub won't let me post a suggestion to a closed pull request, but you can use `empty()` to check if the config value is empty, and you can remove the redundant `else` and just `return` at the end of the function.

Reply:
Hey, thanks! I’ll improve it and reopen a new PR. Thanks for the answer and sorry for pinging you directly!